build/tasks/prepare-release.js

```diff
@@ -36,6 +36,16 @@ gulp.task("publish", (cb) => {
 	return publish("prerelease", "HEAD", cb);
 });
 
+
+gulp.task("prep-release", (cb) => {
```

Comment:
should we just replace the other?

Reply:
first, I left it there, since I thought we are going to leave the old flow. Now I've cleanup all unused tasks.